source/Octopus.Server.Client/Model/EventRetention/EventRetentionConfigurationResource.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Octopus.Client.Extensibility;
+using Octopus.Client.Extensibility.Attributes;
+
+namespace Octopus.Client.Model.EventRetention
+{
+    public class EventRetentionConfigurationResource : IResource
```

Comment:
Any reason to inherit `IResource` over `Resource`?

Reply:
OK. Is `MaintenanceConfigurationResource` an inconsistency? I was using that as a reference

Reply:
Huh, so it does 🙂 I don't know why sorry. 

I can't find any usage of the `LastModified...` fields for `MaintenanceConfigurationResource` (other than a mapper, but that was auto-generated). So I suspect it's not important.